pages/client-libraries/java.mdx

```diff
@@ -4,105 +4,687 @@ description: Peruse our guide on implementing Java in Memgraph. Comprehensive do
+
+#### Simple transaction management  
+
+Simple transaction managment provide a standard blocking style API for Cypher execution. Simple transaction managment provides the easiest programming style to work with since API calls are executed in a strictly sequential fashion.
+
+##### Session management
+
+Before doing any transaction mangment, you need to do is to create a session. Session is a specific connection to the database, that can live for certian time span. Session connection is lightweight and should be closed after the 
+wanted queries has been executed. There is no need to keep the session open for the whole duration of the application. To open a session, use the code snippet below: 
+
+```java
+try (Session session = driver.session(...)) {
+    // transactions go here
+}
+```
+
+ This Java try-with-resources statement will automatically close the session once the code block is finished. Sessions are not thread safe, so you should create a new session for each thread.
+
+With sessions, you can use the following transaction management methods:
+
+ - `writeTransaction` - executes a managed explicit transaction workload in write access mode with retry logic.
+ - `readTransaction` - executes a managed explicit transaction workload in read access mode with retry logic.
+ - `run` - executes a implicit auto-commit transaction and returns result.
+
+
+<Callout type="warning">
+
+Error with multicommand transactions
+
+If you experience `Index manipulation not allowed in multicommand transactions` error,  use [implicit (or auto-commit) transactions](#implicit-transactions) instead.
+
+</Callout>
+
+##### Managed transactions
+
+Let's take a look into the basic example if you want to read data from the database.
+Now you can run simple transaction via `writeTransaction` like this: 
+
+```java
+public void createNode() {
+    try (Session session = driver.session(SessionConfig.forDatabase("memgraph"))) {
+        session.writeTransaction(tx -> {
+            tx.run("CREATE (n:Technology {name:'Memgraph'})");
+            tx.run("CREATE (n:Technology {name:'Java'})");
+            //Some other queries 
+            return null;
+        });
+    }
+}
+```
+As you can see previus examples from guide has used the simple transaction management, it alows you to run multiple queries in a single transaction and have a control over the transaction. You could have multiple logic blocks in the transaction, rollback or end the transaction if needed. 
+Also mange transaction have a retry logic, which means that if the transaction fails, it will be retried. Managed transactions are the most flexible way to run a Cypher query.
```

Comment:
```suggestion
All the previous examples from this guide have used simple transaction management. It allows you to run multiple queries in a single transaction and have control over the transaction. You could have multiple logic blocks in the transaction, rollback or end the transaction if needed. 
Also, managed transactions have a retry logic, which means that if the transaction fails, it will be retried. Managed transactions are the most flexible way to run a Cypher query.
```